tests/SolrIndexSubsitesTest.php

```diff
@@ -1,15 +1,21 @@
 <?php
 
-if (class_exists('Phockito')) {
-    Phockito::include_hamcrest(false);
+namespace SilverStripe\FullTextSearch\Tests;
+
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\FullTextSearch\Tests\SolrIndexSubsitesTest\SolrIndexSubsitesTest_Index;
+
+if (class_exists('\Phockito')) {
+    \Phockito::include_hamcrest(false);
```

Comment:
Ideally we can use this as an opportunity to remove the Phockito dependency. Subsites now has an injectable state service which can be used instead of the global state, but we can also improve the flexibility and extensibility of subsites to allow this example use case to be possible before subsites is tagged as stable.